MetaMask/metamask-extension
# Fix: Actionable Message on Confirm Transaction view is over Tabs

## Explanation

Fixes #14602
Fixes #14932 
Fixes #15271

Also, for context, I saw the Actionable Message has a `z-index: 1` because of https://github.com/MetaMask/metamask-extension/pull/13899, so, in order to avoid disrupting that, I'm just adjusting the tabs. If this is not the best approach, lmk

### Before

<img width="337" alt="Captura de Pantalla 2022-07-18 a la(s) 14 05 16" src="https://user-images.githubusercontent.com/33379285/179511924-eedd328f-3dd5-4912-8dbe-9da0881bbba9.png">

### After

<img width="357" alt="Captura de Pantalla 2022-07-18 a la(s) 14 32 32" src="https://user-images.githubusercontent.com/33379285/179511966-8993bb9c-7516-4f2f-ae28-11037e3dad08.png">

## Manual Testing Steps

Open any page and try to send the following transaction on Rinkeby:

```js
ethereum.request({ method: 'eth_sendTransaction', params: [{"from":"0xbEA76Df6AFccA5E729b839c0A258Df8f359ac64c","to":"0x7e3c49a513d90b35424cf88ea11ecbfef3271c77","data":"0x00"}] })
```

Since it always fail, the message should appear

## Pre-Merge Checklist

- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added (Not sure if applicable)
- [ ] PR has been added to the appropriate release Milestone (Not sure if applicable)
